pyscript.core/src/plugins/error.js

```diff
@@ -24,6 +24,11 @@ hooks.onInterpreterReady.add(function override(pyScript) {
 // Error hook utilities
 
 // Custom function to show notifications
+
+/**
+ * Add a banner to the top of the page, notifying the user of an error
```

Comment:
I think this actually replaces `_createAlertBanner` ... we need some cleanup there.

Reply:
Ah, good to know! I was doing some work with `_createAlertBanner` in another branch (adding warnings for multiple py-configs/config attributes) - I can work on folding that functionality in here.